### PR TITLE
Localizable.string Italian translation

### PR DESCRIPTION
**Pull request by gioelemo**
Translated Localizable.string to Italian

### CODE REVIEW

File: Translations/it.lproj/Localizable.strings [hunk 1]
```diff
@@ -1,10 +1,10 @@
 
 /*Name der App*/
-"app_name" = "Next Step";
+"app_name" = "Prossimo Passo";
```
Comment (darthpelo) [inline Translations/it.lproj/Localizable.strings:3]: Not sure if we need to translate the original name of the application: French and German localization keep "Next Step" as the application name.

File: Translations/it.lproj/Localizable.strings [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
 
 /*App Name unter Icon (iOS)*/
 /*Fuzzy*/
-"CFBundleDisplayName" = "Next Step";
+"CFBundleDisplayName" = "Prossimo Passo";
```
Comment (darthpelo) [inline Translations/it.lproj/Localizable.strings:7]: Same here about the application name

File: Translations/it.lproj/Localizable.strings [hunk 3]
```diff
@@ -156,204 +156,204 @@
...
-"bluetooth_setting_tracking_active_subtext" = "Begegnungen werden anonym auf deinem Gerät gespeichert.";
+"bluetooth_setting_tracking_active_subtext" = "Gli incontri saranno salvati anonimamente su questo dispositivo.";
 
 /*Bluetooth Control: Tracking deaktiviert*/
-"bluetooth_setting_tracking_inactive" = "Tracing deaktiviert";
+"bluetooth_setting_tracking_inactive" = "Tracing disattivato";
 
 /*Bluetooth Control: Tracking deaktiviert Subtext*/
-"bluetooth_setting_tracking_inactive_subtext" = "Erlaube Bluetooth, damit Begegnungen anonym gespeichert werden können.";
+"bluetooth_setting_tracking_inactive_subtext" = "Consenti Bluetooth in modo che gli incontri possano essere salvati in modo anonimo.";
```
Comment (darthpelo) [inline Translations/it.lproj/Localizable.strings:262]: I would suggest:
_Consenti l'utilizzo del Bluetooth_
Apart from that, 👍

**Review by darthpelo — COMMENTED** [review]
Only a couple of notes regarding keep the title of the application in English.
Apart from that, good job.

**Comment by maerki** [comment]
This was implemented on the CH-specific branch and merged back with the latest commit.

**Pull request closed without merging**